platform/binance/client.go

```diff
@@ -16,36 +18,58 @@ type Client struct {
 	blockatlas.Request
 }
 
-func (c *Client) GetBlockList(count int) (*BlockList, error) {
-	result := new(BlockList)
-	query := url.Values{"rows": {strconv.Itoa(count)}, "page": {"1"}}
-	err := c.Get(result, "blocks", query)
+// Fetch runtime information about the node
+func (c *Client) fetchNodeInfo() (*NodeInfo, error) {
+	result := new(NodeInfo)
+	err := c.Get(result, "/v1/node-info", nil)
 	return result, err
 }
 
-func (c *Client) GetBlockByNumber(num int64) (*TxPage, error) {
-	stx := new(TxPage)
+// Get transactions in the block. Multi-send and multi-coin transactions are included as sub-transactions.
+func (c *Client) GetBlockTransactions(num int64) ([]TxV2, error) {
+	stx := new(BlockTxV2)
+	path := fmt.Sprintf("v2/transactions-in-block/%d", num)
+	err := c.Get(stx, path, nil)
+	return stx.Txs, err
+}
+
+//  Gets a list of address or token transactions by type
+func (c *Client) GetAddressAssetTrx(address, token, txType string) ([]Tx, error) {
```

Comment:
do not use `Trx`. Use either `Transactions` or `Tx`, but more intuitive is: `GetAddressAssetTransactions`